lib/Recording/Listener.php

```diff
@@ -34,12 +34,17 @@ public function __construct(
 		protected RecordingService $recordingService,
 		protected ConsentService $consentService,
 		protected IRootFolder $rootFolder,
+		protected LoggerInterface $logger,
 	) {
 	}
 
 	public function handle(Event $event): void {
-		if ($event instanceof AbstractTranscriptionEvent) {
-			$this->handleTranscriptionEvents($event);
+		if ($event instanceof AbstractTaskProcessingEvent) {
+			try {
+				$this->handleTranscriptionEvents($event);
+			} catch (\Throwable $e) {
```

Comment:
Would it make sense to give this process it's own exception?

Reply:
It's not intentional exceptions, I just noticed that a throws exception (due to type error) here killed/removed the re-scheduled TaskProcessing background job.